web/app_options.js

```diff
@@ -210,7 +210,9 @@ const defaultOptions = {
   cMapUrl: {
     /** @type {string} */
     value:
-      typeof PDFJSDev === "undefined" ? "../external/bcmaps/" : "../web/cmaps/",
+      typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")
+        ? "resource://pdf.js/web/cmaps/"
+        : "../external/bcmaps/",
```

Comment:
Here, and below, this is breaking the GENERIC viewer. Please follow the same pattern that you introduced in https://github.com/mozilla/pdf.js/blob/797f8d3dca5e0770c83788b3b65387884031caa0/web/app_options.js#L290-L299